docs/reference/api/estimator.txt

```diff
@@ -47,6 +47,18 @@ API or Native API.
     :members: cache_train_dataset, cache_validation_dataset
     :member-order: bysource
 
+
+Callbacks
+~~~~~~~~~
+
+To execute arbitrary Python functionality during the lifecycle of a
+``EstimatorTrial``, ``determined.estimator.RunHook`` extends
+`tf.estimator.SessionRunHook <https://www.tensorflow.org/api_docs/python/tf/estimator/SessionRunHook/>`_.
+
+.. autoclass:: determined.estimator.RunHook
+    :members: on_checkpoint_load, on_checkpoint_end
+
+
```

Comment:
nonblocking: could be nice to add a code example here making it explicit that these are passed using `build_train_spec` and/or `build_valid_spec`

Reply:
Yep agreed, I added in an example.